src/RpcClient/Utility.cs

```diff
@@ -236,7 +236,7 @@ public static StackItem StackItemFromJson(JObject json)
                 case StackItemType.InteropInterface:
                     return new InteropInterface(new object()); // See https://github.com/neo-project/neo/blob/master/src/neo/VM/Helper.cs#L194
             }
-            return null;
+            return json["value"] is null ? StackItem.Null : json["value"].AsString();
```

Comment:
`{"type":"myType"}` should fail

Reply:
Because StackItemType does not support custom types.

https://github.com/neo-project/neo-modules/blob/75a78575752f3d55a7fba004d240b3bbc961dbcf/src/RpcClient/Utility.cs#L205

This line result: `StackItemType.Any`